lingua_franca/res/text/az-az/date_time.json

```diff
@@ -0,0 +1,130 @@
+{
+  "decade_format": {
+    "1": {"match": "^\\d$", "format": "{x}"},
+    "2": {"match": "^1\\d$", "format": "{xx}"},
+    "3": {"match": "^\\d0$", "format": "{x0}"},
+    "4": {"match": "^[2-9]\\d$", "format": "{x0} {x}"},
+    "default": "{number}"
+  },
+    "hundreds_format": {
+      "1": {"match": "^1\\d{2}$", "format": "yüz"},
+      "2": {"match": "^[2-9]\\d{2}$", "format": "{x_in_x00} yüz"},
+      "default": "{number}"
+  },
+    "thousand_format": {
+      "1": {"match": "^10\\d{2}$", "format": "min"},
+      "2": {"match": "^[2-9]0\\d{2}$", "format": "{x_in_x000} min"},
+      "3": {"match": "^11\\d{2}$", "format": "min yüz"},
+      "4": {"match": "^1\\d{3}$", "format": "min {x_in_x00} yüz"},
+      "5": {"match": "^[2-9]1\\d{2}$", "format": "{x_in_x000} min yüz"},
+      "6": {"match": "^[2-9]\\d{3}$", "format": "{x_in_x000} min {x_in_x00} yüz"},
+      "default": "{number}"
+  },
+    "year_format": {
+      "1": {"match": "^\\d{1}$", "format": "{bc} {formatted_decade}"},
+      "2": {"match": "^\\d{2}$", "format": "{bc} {formatted_decade}"},
+      "3": {"match": "^\\d00$", "format": "{bc} {formatted_hundreds}"},
+      "4": {"match": "^\\d{3}$", "format": "{bc} {formatted_hundreds} {formatted_decade}"},
+      "5": {"match": "^\\d\\d00$", "format": "{bc} {formatted_thousand}"},
+      "6": {"match": "^\\d{4}$", "format": "{bc} {formatted_thousand} {formatted_decade}"},
+      "default": "{bc} {year}",
+      "bc": "e.ə."
+  },
+    "date_format": {
+      "date_full": "{weekday}, {month} {day}, {formatted_year}",
+      "date_full_no_year": "{weekday}, {month} {day}",
+      "date_full_no_year_month": "{weekday}, ayın {day}",
+      "today": "bu gün",
+      "tomorrow": "sabah",
+      "yesterday": "dünən"
+  },
+    "date_time_format": {
+      "date_time": "{formatted_date} {formatted_time}"
+  },
+    "weekday": {
+    "0": "bazar ertəsi",
+    "1": "çərşənbə axşamı",
+    "2": "çərşənbə",
+    "3": "cümə axşamı",
+    "4": "cümə",
+    "5": "şənbə",
+    "6": "bazar"
```

Comment:
I'm guessing "bazar" is used in informal speech rather than "bazar günü"?

Reply:
No, the equivalent for sunday is "bazar", "bazar günü" means "on sunday" and is the same other days, "şənbə günu" is "on saturday",...

Reply:
and this is why we don't use automated translation tools for this stuff :smile: